packages/fcl/src/discovery/services.js

```diff
@@ -1,6 +1,29 @@
 import {config} from "@onflow/sdk"
 import {invariant} from "@onflow/util-invariant"
 
+const SERVICES_PLATFORMS = {
+  EXTENSION: "web/extension"
+}
+
+function deepEqual(object1, object2) {
```

Comment:
Why a deep equals? Do the services need to match exactly?

Reply:
So someone can't inject a service with different values. (e.g. "name")